### DISCUSSION TITLE
Chaos experiments injecting faults to different clusters.

### DISCUSSION THREAD

**Discussion by ramu-brao** [chaos-mesh/chaos-mesh#2362] [Q&A]
Hello Team,

I have one requirement. I had my chaos dashboard and things running in one Kubernetes cluster(EKS) which is like sandbox, I also have other cluster called Dev cluster without any stuff related to chaos mesh. So my question is, having chaos dashboard in cluster and apply or inject faults on the pods which is running in other cluster as well? So my chaos stuff resides in only one cluster and should be able to use this for injecting faults across all the other clusters. Is there any possible way of doing this?

**Comment by YangKeao** [comment]
It's not possible now. But @WangXiangUSTC and @iguoyr is trying to design and implement this feature :+1: .

**Reply by angoothachap** [reply]
Is there an Issue I can follow? Thanks!

**Reply by iguoyr** [reply]
Hi @angoothachap, @YangKeao is working on this feature now, the tracking issue is #3341. You could read [this rfc](https://github.com/YangKeao/rfcs/blob/multi-cluster-support/text/2022-05-12-multi-cluster-support.md) first, and feel free to discuss if you have any ideas.
